src/Entity/Contact.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Ramsey\Uuid\Uuid;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity()
+ */
+class Contact
+{
+    /**
+     * @ORM\Id()
+     * @ORM\Column(type="guid")
+     */
+    private $id;
+
+    /**
+     * @Assert\NotBlank()
+     * @ORM\Column(type="string", length=63)
+     */
+    private $firstName;
+
+    /**
+     * @ORM\Column(type="string", length=63, nullable=true)
+     */
+    private $lastName;
+
+    /**
+     * @Assert\Email()
+     * @ORM\Column(type="string", length=127, nullable=true)
```

Comment:
this might be too short, email can go to 256 characters with the actual standard RFC 2821